app/livechat/client/views/app/tabbar/customerChatHistoryMessages.js

```diff
@@ -0,0 +1,101 @@
+import { ReactiveVar } from 'meteor/reactive-var';
+import { Template } from 'meteor/templating';
+import moment from 'moment';
+import _ from 'underscore';
+import './customerChatHistoryMessages.html';
+import { APIClient, t } from '../../../../../utils/client';
+import { AccountBox, TabBar, MessageTypes } from '../../../../../ui-utils';
+import {Mongo ,MongoInternals} from 'meteor/mongo';
+import { Tracker } from 'meteor/tracker';
+import { ReactiveDict } from 'meteor/reactive-dict';
+import { Session } from 'meteor/session';
+
+let  historyResult 
+let len, msgs
+let allHistoryChat ;
+
+Template.customerChatHistoryMessages.helpers({
+    
+	historyResult(){
+		// will return all the messages in history room
+		
+		return Template.instance().historyResu.get().reverse();
+    },
+    len(){
```

Comment:
You don't need this.

Reply:
i need to reverse the array cause of it is because recent one coming first